components/eamxx/src/share/iop/intensive_observation_period.cpp

```diff
@@ -541,6 +541,15 @@ read_iop_file_data (const util::TimeStamp& current_ts)
     scorpio::read_var(iop_file,"Ps",ps_data,iop_file_time_idx);
     surface_pressure.sync_to_dev();
 
+    // Read in IOP lev data
+    auto data = iop_file_pressure.get_view<Real*, Host>().data();
+    scorpio::read_var(iop_file,"lev",data);
+
+    // Convert to pressure to millibar (file gives pressure in Pa)
+    for (int ilev=0; ilev<file_levs; ++ilev) data[ilev] /= 100;
+    iop_file_pressure.sync_to_dev();
+    m_helper_fields.insert({"iop_file_pressure", iop_file_pressure});
```

Comment:
Remove this `insert()` call. It will do nothing since the key "iop_file_pressure" already exists in the `m_helper_fields` map object. And we are updating the data of the field stored in the map, so all the info is automatically in the map object when we update the data.